include/rcutils/error_handling.h

```diff
@@ -39,6 +41,8 @@ typedef struct rcutils_error_state_t
   rcutils_allocator_t allocator;
 } rcutils_error_state_t;
 
+#define SAFE_FWRITE_TO_STDERR(msg) fwrite(msg, sizeof(char), sizeof(msg), stderr)
```

Comment:
Since this is a public header, this needs to be namespaced, e.g. `RCUTILS_SAFE_FWRITE_TO_STDERR` or something like that.